test_batch_sim acts flaky on github actions

____________ BatchSimulatorTest.test_batch_ctx_status_failing_jobs _____________

self = <test_batch_sim.BatchSimulatorTest testMethod=test_batch_ctx_status_failing_jobs>

    @tmpdir()
    def test_batch_ctx_status_failing_jobs(self):
    
        config_dir = os.path.join(self.TESTDATA_ROOT, "local/batch_sim")
        shutil.copytree(config_dir, "batch_sim")
        os.chdir("batch_sim")
    
        self.assertTrue(os.path.isfile("batch_sim_sleep_and_fail.ert"))
        res_config = ResConfig(user_config_file="batch_sim_sleep_and_fail.ert")
    
        external_parameters = {
            "WELL_ORDER": ("W1", "W2", "W3"),
            "WELL_ON_OFF": ("W1", "W2", "W3"),
        }
        results = ("ORDER", "ON_OFF")
        rsim = BatchSimulator(res_config, external_parameters, results)
    
        cases = [
            (
                0,
                {
                    "WELL_ORDER": {"W1": idx + 1, "W2": idx + 2, "W3": idx + 3},
                    "WELL_ON_OFF": {"W1": idx * 4, "W2": idx * 5, "W3": idx * 6},
                },
            )
            for idx in range(10)
        ]
    
        batch_ctx = rsim.start("case_name", cases)
        while batch_ctx.running():
>           self.assertContextStatusOddFailures(batch_ctx)

/home/runner/work/ert/ert/tests/libres_tests/res/simulator/test_batch_sim.py:595: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/home/runner/work/ert/ert/tests/libres_tests/res/simulator/test_batch_sim.py:563: in assertContextStatusOddFailures
    self.assertEqual(JobStatusType.JOB_QUEUE_FAILED, status)
E   AssertionError: JobStatusType(name = "JOB_QUEUE_FAILED", value = 8192) != None

got much of the same: 
```
____________ BatchSimulatorTest.test_batch_ctx_status_failing_jobs _____________

self = <test_batch_sim.BatchSimulatorTest testMethod=test_batch_ctx_status_failing_jobs>

    @tmpdir()
    def test_batch_ctx_status_failing_jobs(self):
    
        config_dir = os.path.join(self.TESTDATA_ROOT, "local/batch_sim")
        shutil.copytree(config_dir, "batch_sim")
        os.chdir("batch_sim")
    
        self.assertTrue(os.path.isfile("batch_sim_sleep_and_fail.ert"))
        res_config = ResConfig(user_config_file="batch_sim_sleep_and_fail.ert")
    
        external_parameters = {
            "WELL_ORDER": ("W1", "W2", "W3"),
            "WELL_ON_OFF": ("W1", "W2", "W3"),
        }
        results = ("ORDER", "ON_OFF")
        rsim = BatchSimulator(res_config, external_parameters, results)
    
        cases = [
            (
                0,
                {
                    "WELL_ORDER": {"W1": idx + 1, "W2": idx + 2, "W3": idx + 3},
                    "WELL_ON_OFF": {"W1": idx * 4, "W2": idx * 5, "W3": idx * 6},
                },
            )
            for idx in range(10)
        ]
    
        batch_ctx = rsim.start("case_name", cases)
        while batch_ctx.running():
>           self.assertContextStatusOddFailures(batch_ctx)

/home/runner/work/ert/ert/tests/libres_tests/res/simulator/test_batch_sim.py:595: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/home/runner/work/ert/ert/tests/libres_tests/res/simulator/test_batch_sim.py:561: in assertContextStatusOddFailures
    self.assertEqual(JobStatusType.JOB_QUEUE_SUCCESS, status)
E   AssertionError: JobStatusType(name = "JOB_QUEUE_SUCCESS", value = 512) != None
=============================== warnings summary ===============================
```

--- a/tests/libres_tests/res/simulator/test_batch_sim.py
+++ b/tests/libres_tests/res/simulator/test_batch_sim.py
@@ -550,6 +550,7 @@ class BatchSimulatorTest(ResTest):
                 JobStatusType.JOB_QUEUE_UNKNOWN,
                 JobStatusType.JOB_QUEUE_EXIT,
                 JobStatusType.JOB_QUEUE_DONE,
+                None,  # job is not submitted yet but ok for this test
             )
         )
 

